shell/components/ExplorerMembers.vue

```diff
@@ -238,6 +304,14 @@ export default {
     getProjectLabel(group) {
       return this.getMgmtProject(group)?.spec?.displayName;
     },
+    getProjectPermissions(projectId) {
+      return this.currentUsersProjectPermissions[projectId] || {};
+    },
+    getManageProjectMembersPermission(projectId) {
+      const projectRoleTemplateBindingPermissions = this.getProjectPermissions(projectId)?.['management.cattle.io']?.['projectroletemplatebindings'];
+
+      return projectRoleTemplateBindingPermissions.some(verb => ['*', 'own', 'create'].includes(verb));
```

Comment:
page fails to load for admin users when there is a project that they don't have a role in. this results in projectRoleTemplateBindingPermissions being undefined.

Making this null safe then hides the cross & add buttons for admin users.

This is why https://github.com/rancher/dashboard/pull/9100#pullrequestreview-1479669501 stated admin OR other cases. admins then also don't need to go through the complicated reduce above